whole-tale/ngx-dashboard
# fix: remove obsolete Workspace service

## Problem

Drops dead code related to now obsoleted backend path.

## How to Test

1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. Play around, nothing should be broken.
